docs/index.md

```diff
@@ -107,12 +104,12 @@ class YourCoreLib(CoreLib):
         ...
 ```
 #### `your_core_lib.py` Explained:
-**In your_core_lib.py, a `custom CoreLib class` and a `SqlAlchemyConnectionRegistry class` are defined to manage database connections.**
+**In `your_core_lib.py`, a `custom CoreLib class` and a `SqlAlchemyConnectionRegistry class` are defined to manage database connections.**
 
 #### Defining a new class YourCoreLib that inherits from CoreLib.
-  - Defining an __init__ method for YourCoreLib that takes a config argument of type [DictConfig](https://omegaconf.readthedocs.io/en/2.3_branch/api_reference.html#id1){:target="_blank"}. It is a dictionary type from [omegaconf](https://omegaconf.readthedocs.io/en/2.3_branch/index.html){:target="_blank"} that used by [Hydra](https://hydra.cc/docs/intro/){:target="_blank"}.
+  - Defining an `__init__` method for `YourCoreLib` that takes a config argument of type [DictConfig](https://omegaconf.readthedocs.io/en/2.3_branch/api_reference.html#id1){:target="_blank"}. It is a dictionary type from [omegaconf](https://omegaconf.readthedocs.io/en/2.3_branch/index.html){:target="_blank"} that used by [Hydra](https://hydra.cc/docs/intro/){:target="_blank"}.
   - Calling the parent class CoreLib's `__init__` method using `CoreLib.__init__(self)` to initialize the base class.
-    - Mark core-lib started
+    - Mark core-lib as started
     - Enable the use of core-lib observers
   - Initialize a `db_connection` object by instantiating `SqlAlchemyConnectionRegistry` with the SQLAlchemy configuration fetched from `self.config.core_lib.data.sqlalchemy`, thereby connecting to and managing the specified database as defined in the `your_core_lib.yaml` configuration file.
   - Initializing a `UserDataAccess` class responsible for accessing user data from the database, passing the `db_connection` object to it.
```

Comment:
in def main(cfg : DictConfig) missing the type,. make sure to add the type everywhere